mailinglists.md

```diff
@@ -13,5 +13,17 @@ All communication related to the Apache Drill project happens on the mailing lis
   * Subscribe: <dev-subscribe@drill.apache.org>
   * Unsubscribe: <dev-unsubscribe@drill.apache.org>
   * Post: <dev@drill.apache.org>
-  * [Archive](http://mail-archives.apache.org/mod_mbox/drill-dev/)
+  * [Archive](http://mail-archives.apache.org/mod_mbox/drill-dev/)  
+
+**Note:** All contributions to the Apache projects are made by individual contributors, not organizations. Though companies may be paying their staff to commit code, promote the project, and so on, contributions are recognized on an individual basis. The ASF is vendor-neutral.
+ 
+When communicating through project channels, please follow these general guidelines:  
+
+* Be courteous and respectful.
+* Be concise in asking your question. Before doing so, try to read available documentation, FAQs, and mailing list history.
+* Provide enough technical background to explain the issue.
+
+For additional guidelines, see [How To Ask Questions The Smart Way](http://www.catb.org/esr/faqs/smart-questions.html), and note the disclaimer:  
```

Comment:
I would not quote the disclaimer. Perhaps change it to -
" and note the disclaimer (so please do not ask Drill questions on catb.org). "